.prow.yaml

```diff
@@ -1283,16 +1283,28 @@ presubmits:
     clone_uri: "ssh://git@github.com/kubermatic/kubermatic.git"
     labels:
       preset-openstack: "true"
+      preset-vsphere: "true"
       preset-docker-pull: "true"
       preset-docker-push: "true"
+      preset-vault: "true"
+      preset-repo-ssh: "true"
+      preset-e2e-ssh: "true"
       preset-kind-volume-mounts: "true"
+      preset-kubeconfig-ci: "true"
       preset-goproxy: "true"
     spec:
       containers:
         - image: quay.io/kubermatic/build:go-1.17-node-16-kind-0.11-5
           env:
             - name: KUBERMATIC_EDITION
               value: ce
+            - name: DISTRIBUTIONS
+              value: ubuntu
+            - name: SERVICE_ACCOUNT_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: e2e-ci
+                  key: serviceAccountSigningKey
```

Comment:
Would it make sense that we split this job into two — one for OpenStack and one for vSphere? For example, if vSphere goes down for an extended time, the whole job would fail even if OpenStack tests are passing. We would have to figure out how to modify the code to disable vSphere tests, which would probably take more time than disabling a Prow job.